airbyte-cdk/python/unit_tests/sources/declarative/requesters/request_options/test_interpolated_request_options_provider.py

```diff
@@ -45,12 +50,17 @@ def test_interpolated_request_params(test_name, input_request_params, expected_r
         ("test_value_depends_on_stream_slice", {"read_from_slice": "{{ stream_slice['start_date'] }}"}, {"read_from_slice": "2020-01-01"}),
         ("test_value_depends_on_next_page_token", {"read_from_token": "{{ next_page_token['offset'] }}"}, {"read_from_token": 12345}),
         ("test_value_depends_on_config", {"read_from_config": "{{ config['option'] }}"}, {"read_from_config": "OPTION"}),
-        ("test_none_value", {"missing_json": "{{ fake_path['date'] }}"}, {}),
+        ("test_missing_value", {"missing_json": "{{ fake_path['date'] }}"}, {"missing_json": None}),
         (
             "test_interpolated_keys",
             {"{{ stream_state['date'] }}": 123, "{{ config['option'] }}": "ABC"},
             {"2021-01-01": 123, "OPTION": "ABC"},
         ),
+        ("test_boolean_false_value", {"boolean_false": "{{ False }}"}, {"boolean_false": False}),
+        ("test_integer_falsy_value", {"integer_falsy": "{{ 0 }}"}, {"integer_falsy": 0}),
+        ("test_number_falsy_value", {"number_falsy": "{{ 0.0 }}"}, {"number_falsy": 0.0}),
+        ("test_string_falsy_value", {"string_falsy": "{{ '' }}"}, {"string_falsy": None}),
+        ("test_none_value", {"none_value": "{{ None }}"}, {"none_value": None}),
```

Comment:
when would we want to return `None` in the request body?

Reply:
In theory I can imagine something like this:

```
curl -d '{"key1":null}' -X POST http://api.com/api
```

But to be honest we did not encounter in this for now. For now we need only this:

```
curl -d '{"key1":false}' -X POST http://api.com/api
```

I have re-implemented doing less radical changes
What do you think now?